versioned_docs/version-5/migration.mdx

```diff
@@ -695,6 +695,145 @@ actor.subscribe((state) => {
 
 ## Actors
 
+### Use actor logic creators for `invoke.src` instead of functions
+
+:::breakingchange
+
+Breaking change
+
+:::
+
+The available actor logic creators are:
+
+- `createMachine`
+- `fromPromise`
+- `fromObservable`
+- `fromEventObservable`
+- `fromTransition`
```

Comment:
Would we soon add examples for `fromEventObservable` and `fromTransition`? I think people would want to see those. And do we have good v4 "examples" to compare to in the tabbed interface?

Also, once we add all examples it could be nice to separate the tabbed examples with smaller headers and a single line describing what they're about to see in the example, instead of just 5 tabbed examples one after the other where you need to read the imports or the code to find the one you want.